src/bundle/Resources/translations/messages.en.xliff

```diff
@@ -6,6 +6,11 @@
       <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
     </header>
     <body>
+      <trans-unit id="da39a3ee5e6b4b0d3255bfef95601890afd80709" resname="base.footer_copyright">
+        <source>eZ Platform ©1999-%current_year% %ez_link_open%eZ Systems AS%ez_link_close% and others.</source>
+        <target state="new">eZ Platform ©1999-%current_year% %ez_link_open%eZ Systems AS%ez_link_close% and others.</target>
+        <note>key: </note>
```

Comment:
Don't know if it matters but the key name is missing here.

Reply:
Oh, thank you. Regenerating the new string didn't work, and I ended up updating it manually. I'll add it.

Reply:
Added the missing key, fixed up and rebased.